packages/design-system/src/components/disclosure/index.js

```diff
@@ -25,6 +25,7 @@ import PropTypes from 'prop-types';
 import { THEME_CONSTANTS } from '../../theme';
 import { ChevronDownSmall } from '../../icons';
 
+// TODO: Account for RTL layout for 'left' and 'right'?
```

Comment:
Note: I'm still reviewing the code here and haven't looked at it to do a quick QA, but do you see any issues with how this works/looks before and after enabling RTL?

Reply:
Thinking further, this component should not handle RTL layout.